egs/chime4/s5_1ch/local/compute_stoi_estoi_sdr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
+# Apache 2.0
```

Comment:
some explanation here would be nice.  The filename is very obscure; what is the script for?

Reply:
Added documentation and example usage below the license.